subresource-integrity/idlharness.window.js

```diff
@@ -1,17 +0,0 @@
-// META: script=/resources/WebIDLParser.js
-// META: script=/resources/idlharness.js
-
-// https://w3c.github.io/webappsec-subresource-integrity/
```

Comment:
Did you really mean to delete this test?

Reply:
Yes; it's contents have been bundled into the HTML spec, so it's failing because of the member clash.

Reply:
Aha, thanks.